tibetegya/Cartale_api
# add express server starter server

#### What does this PR do?
This PR adds the express server boilerplate code
#### Description of Task to be completed?
- Setup express server
#### How should this be manually tested?
- git clone this REPO
- `cd Cartale_api`
- `npm install`
- Create a .env file if you don't have one and in your .env file add `PORT=3030`
- `npm run start:dev`
-  In your browser navigate to `localhost:3030` it should render the welcome to express page
